### PR TITLE
migrate: aws v2 migration ses + security hub

### PR DESCRIPTION
**Pull request by james03160927**
## Description

Partially fixing https://github.com/gruntwork-io/cloud-nuke/issues/770. 

## TODOs

Read the [Gruntwork contribution guidelines](https://gruntwork.notion.site/Gruntwork-Coding-Methodology-02fdcd6e4b004e818553684760bf691e).

- [ ] Update the docs.
- [ ] Run the relevant tests successfully, including pre-commit checks.
- [ ] Ensure any 3rd party code adheres with our [license policy](https://www.notion.so/gruntwork/Gruntwork-licenses-and-open-source-usage-policy-f7dece1f780341c7b69c1763f22b1378) or delete this line if its not applicable.
- [ ] Include release notes. If this PR is backward incompatible, include a migration guide.
- [ ] Attention Grunts - if this PR adds support for a new resource, ensure the `nuke_sandbox` and `nuke_phxdevops` jobs in `.circleci/config.yml` have been updated with appropriate exclusions (either directly in the job or via the `.circleci/nuke_config.yml` file) to prevent nuking IAM roles, groups, resources, etc that are important for the test accounts.

## Release Notes (draft)

Added / Removed / Updated [X].

### CODE REVIEW

File: aws/resources/security_hub.go [hunk 1]
```diff
@@ -114,33 +113,33 @@ func (sh *SecurityHub) nukeAll(securityHubArns []string) error {
...
 	// Check for an administrator account
 	// Security hub cannot be disabled with an active administrator account
-	adminAccount, err := sh.Client.GetAdministratorAccountWithContext(sh.Context, &securityhub.GetAdministratorAccountInput{})
+	adminAccount, err := sh.Client.GetAdministratorAccount(sh.Context, &securityhub.GetAdministratorAccountInput{})
 	if err != nil {
 		logging.Errorf("[Failed] Failed to check for administrator account")
 	}
 
 	// Disassociate administrator account if it exists
 	if adminAccount.Administrator != nil {
```
Comment (denis256) [inline aws/resources/security_hub.go:122]: hm, `adminAccount` can be null? looks like after `err` handling execution continue with checking `adminAccount` fields

**Review by denis256 — COMMENTED** [review]
Conflicting files
v2_migration_report/output.md

**Comment by james03160927** [comment]
Hi @denis256, this should be good for review again! PTAL.

**Comment by denis256** [comment]
Looks like build failed:
```
aws/resources/security_hub.go:8:2: no required module provides package github.com/aws/aws-sdk-go-v2/service/securityhub; to add it:
	go get github.com/aws/aws-sdk-go-v2/service/securityhub
aws/resources/ses_configuration_set.go:7:2: no required module provides package github.com/aws/aws-sdk-go-v2/service/ses; to add it:
	go get github.com/aws/aws-sdk-go-v2/service/ses
```

```
2 errors occurred:
--> windows/386 error: exit status 1
Stderr: aws/resources/security_hub.go:8:2: no required module provides package github.com/aws/aws-sdk-go-v2/service/securityhub; to add it:
	go get github.com/aws/aws-sdk-go-v2/service/securityhub
aws/resources/ses_configuration_set.go:7:2: no required module provides package github.com/aws/aws-sdk-go-v2/service/ses; to add it:
	go get github.com/aws/aws-sdk-go-v2/service/ses

--> windows/386 error: exit status 1
Stderr: aws/resources/security_hub.go:8:2: no required module provides package github.com/aws/aws-sdk-go-v2/service/securityhub; to add it:
	go get github.com/aws/aws-sdk-go-v2/service/securityhub
aws/resources/ses_configuration_set.go:7:2: no required module provides package github.com/aws/aws-sdk-go-v2/service/ses; to add it:
	go get github.com/aws/aws-sdk-go-v2/service/ses
```

**Comment by james03160927** [comment]
> Looks like build failed:
> 
> ```
> aws/resources/security_hub.go:8:2: no required module provides package github.com/aws/aws-sdk-go-v2/service/securityhub; to add it:
> 	go get github.com/aws/aws-sdk-go-v2/service/securityhub
> aws/resources/ses_configuration_set.go:7:2: no required module provides package github.com/aws/aws-sdk-go-v2/service/ses; to add it:
> 	go get github.com/aws/aws-sdk-go-v2/service/ses
> ```
> 
> ```
> 2 errors occurred:
> --> windows/386 error: exit status 1
> Stderr: aws/resources/security_hub.go:8:2: no required module provides package github.com/aws/aws-sdk-go-v2/service/securityhub; to add it:
> 	go get github.com/aws/aws-sdk-go-v2/service/securityhub
> aws/resources/ses_configuration_set.go:7:2: no required module provides package github.com/aws/aws-sdk-go-v2/service/ses; to add it:
> 	go get github.com/aws/aws-sdk-go-v2/service/ses
> 
> --> windows/386 error: exit status 1
> Stderr: aws/resources/security_hub.go:8:2: no required module provides package github.com/aws/aws-sdk-go-v2/service/securityhub; to add it:
> 	go get github.com/aws/aws-sdk-go-v2/service/securityhub
> aws/resources/ses_configuration_set.go:7:2: no required module provides package github.com/aws/aws-sdk-go-v2/service/ses; to add it:
> 	go get github.com/aws/aws-sdk-go-v2/service/ses
> ```

Hmm it seems like some of the files were not included in the PR. PTAL again.

**Review by denis256 — APPROVED** [review]

**Pull request merged**